shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/migration/MigrationJobPreparer.java

```diff
@@ -143,8 +145,12 @@ private void prepareTarget(final MigrationJobItemContext jobItemContext) throws
     }
     
     private void initInventoryTasks(final MigrationJobItemContext jobItemContext) {
+        InventoryDumperConfiguration inventoryDumperConfiguration = new InventoryDumperConfiguration(jobItemContext.getTaskConfig().getDumperConfig());
```

Comment:
`inventoryDumperConfiguration` could be `inventoryDumperConfig` or `dumperConfig`